src/components/InfoViewer/InfoViewer.tsx

```diff
@@ -15,6 +15,7 @@ interface InfoViewerProps {
     size?: 's';
     className?: string;
     multilineLabels?: boolean;
+    renderOnEmptyState?: () => JSX.Element | null;
```

Comment:
i believe the suggested prop name was `renderEmptyState` (not `...On...`)

Reply:
it should also just return `ReactNode`, which includes null, and will allow a simple string